packages/foam-vscode/src/core/model/workspace.ts

```diff
@@ -57,7 +62,11 @@ export class FoamWorkspace implements IDisposable {
   }
 
   public resources(): IterableIterator<Resource> {
-    return this._resources.values();
+    const resources: Array<Resource> = Array.from(
+      this._resources.values()
+    ).sort(Resource.sortByPath);
+
+    return resources.values();
```

Comment:
do we need the `.values()` again on this line?

Reply:
It seems so. The only way I can see to return an IterableIterator